src/views/Unauthorized/Unauthorized.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <div>
+    <PageTitle :description="description" />
+  </div>
+</template>
+<script>
+import PageTitle from '../../components/Global/PageTitle';
+export default {
+  name: 'unauthorized',
```

Comment:
Sorry missed this initially and only recently started noticing our inconsistencies with casing for component names. I think according to the Vue style guide PascalCase is recommended https://vuejs.org/v2/style-guide/#Component-name-casing-in-JS-JSX-strongly-recommended 
Thoughts?

Reply:
Actually, that brings up something that I have been meaning to discuss. If we look at our .eslintrc file and change the from `plugin:vue/essential'` to `plugin:vue/strongly-recommended`, we have some work to do. I think we should make a decision on the standard now. 

My feeling is that we should use `recommended`, but I'm interested in hearing what others think. 
- Essential is great because it is focused on preventing errors. 
- Strongly-recommended focuses on readability. 
- Recommended removes arbitrary choices

Here is the complete list: https://eslint.vuejs.org/rules/

If you update the .eslintrc.js file and take a look in VSCode you quickly see we have some work to do. :)

Reply:
This is an example of areas we would need to address. This is the ModalSettings.vue file in LocalUserManagement

![image](https://user-images.githubusercontent.com/1183515/71748551-86a18500-2e38-11ea-8997-6b0e921dc021.png)

Reply:
```
Module Warning (from ./node_modules/eslint-loader/index.js):
warning: Prop "sectionTitle" should define at least its type (vue/require-prop-types) at src/components/Global/PageSection.vue:11:11:
   9 | export default {
  10 |   name: 'PageSection',
> 11 |   props: ['sectionTitle']
     |           ^
  12 | };
  13 | </script>
  14 |
```

https://vuejs.org/v2/guide/components-props.html#Prop-Types

Reply:
For now, I will update to use PascalCase though. Thank you for catching this @yoshiemuranaka! I wonder where we should continue this conversation. What do you think or a new story that would include the discussion about the decision and then updating of all the files?

Reply:
Story added and I updated and pushed to use PascalCase.

Story: https://github.com/gtmills/phosphor-webui-vue/issues/71